### ISSUE TITLE
Allow MatchTaxatoEOLID to handle multiple search results for a single EOL ID

### ISSUE THREAD

**Issue by tdjames1** [ropensci/reol#9]
Sometimes the EOL API returns multiple results for taxonomic name associated with a single EOL ID, 

e.g. http://eol.org/api/search/1.0.xml?q=Turdus%20merula&exact=TRUE&page=1

Currently, `MatchTaxatoEOLID` returns NA for a taxa if more than one search result is returned by the EOL API. 

    MatchTaxatoEOLID("Turdus merula")

It would be good to check whether multiple results are all associated with the same EOL ID, and return the EOL ID information in that case.

**Comment by sckott** [comment]
Any thoughts on this @bbanbury ?

**Comment by tdjames1** [comment]
I should add that I do have a workaround for this that I was using as a hotfix when I was using Reol for a project a while back. I'll submit a pull request later.

**Comment by bbanbury** [comment]
Hi all, I will not be able to get to this before the weekend. Looks like the api has changed a bit since the code base was written. My examples no longer return a taxon either. If you have a work-around and want to submit a pull, that works for me.

**Comment by tdjames1** [comment]
Yes, I noticed that some of the examples in the vignette don't work properly now. There's a related issue but separate issue where the API returns multiple matches for a taxon because some trait data related to that species has been added, but not validated, so it has been assigned a different EOL ID. My workaround only handles the case when multiple entries with the same EOL ID are returned from a search.
